Formula/proteinortho.rb

```diff
@@ -0,0 +1,22 @@
+class Proteinortho < Formula
+  desc "Proteinortho is a tool to detect orthologous genes within different species"
+  homepage "https://gitlab.com/paulklemm_PHD/proteinortho"
+  url "https://gitlab.com/paulklemm_PHD/proteinortho/-/archive/v6.0b/proteinortho-v6.0b.tar.gz"
+  sha256 "f59c436f6461f318ccebc1177a67eb70c29ef9ba222ad724de1fb2a769a120b5"
+  depends_on "cmake" => :build
+  depends_on "gcc"
+
+  # get the latest g++ compiler from brew (Cellar)
+  CXX = `ls /usr/local/Cellar/gcc/*/bin/g++* | sort | tail -n1 | tr -d '\n'`
```

Comment:
this is not okay, the `/usr/local` install path isn't guaranteed and neither is the shell. You can use an `ENV.cxx11` call or something similar if you need a specific version.

Reply:
ahhh ok i was searching for that, i will fix that

Reply:
So i tried `ENV.cxx11` but it does not seem to have any effect (the g++ compiler is still the apple one and not the Cellar/brew g++)
After calling `ENV.cxx11` what happens

```
  def install
    ENV.cxx11
    system "mkdir", "#{prefix}/bin"
    system "make", "install", "PREFIX=#{prefix}/bin"
  end
```
Then it fails because apples g++ does not have gfortran...
Any clue what i am doing wrong? 
(I looked at /usr/bin/g++ and it is still the same old clang g++ and $CXX points to clang++, that is not what i need)

Reply:
Here's a formula with the same problem (needing gfortran) https://github.com/Homebrew/homebrew-core/blob/master/Formula/octave.rb

Reply:
Similar to octave i have now this solution:
```
  def install
    system "mkdir", "#{prefix}/bin"
    ENV["CXX"] = `ls #{Formula["gcc"].opt_bin.realpath}/g++-* | sort | tail -n1`.chomp
    system "make", "CPP=$CXX" , "install", "PREFIX=#{prefix}/bin"
  end
```
Would that be ok?